app/assets/stylesheets/sulFooter.scss

```diff
@@ -0,0 +1,40 @@
+$brand-color: $color-cardinal-red !default;
+$footer-color: $white !default;
+$pre-footer-background: $fog-light !default;
+
+// These body/main styles allow the SUL and SU footers to render at the bottom of the viewport
+body {
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+}
+
+main {
+  flex: 1 0 auto;
+  @extend .mt-4;
+}
```

Comment:
See comment. Without this, the footers are broken. This snippet comes from vt-arclight, the latest project to adapt the Decanter styles for the SU global footer.